### DISCUSSION TITLE
Google Analytics 4

### DISCUSSION THREAD

**Discussion by celsobtu** [AzuraCast/AzuraCast#6007] [General]
Hello friends, could someone please let me know if AzuraCast in version v0.17.6 Stable already works with Google Analytis 4 via web Hooks? It's not working for me, what do I need to do to make it work?
tks
Celso

**Comment by BusterNeece** [comment]
Update: I looked into it, and it turns out there *is* a Google Analytics Measurement Protocol library that we can roll out, so the newest Rolling Release includes GA V4 support.

**Reply by celsobtu** [reply]
@BusterNeece  .... Great ! this is one
great news and will help a lot.
The current GA-Universal will be deactivated from 01/July/23
Thank you very much
